scripts/seastar-json2code.py

```diff
@@ -256,68 +257,120 @@ def resolve_model_order(data):
             models.add(model_name)
     return res
 
+
+def not_first():
+    '''
+    Returns True when gets called for the first time, False otherwise
+
+    used as the predicate parameter of textwrap.indent(), so the first
+    line is not indented. this helps to preserve the Python code's logical
+    indention in the template, and allows us to put something like::
+
+      blah = textwrap.dedent("""\
```

Comment:
With the backslash, it's not needed, since we don't have an empty first line.

Reply:
this is needed for the cases of 

```py
    switch(v) {
    $case_clauses
    }
```
where `case_clauses` is composed like `"\n".join("case foo: return bar;")`. so in this case, we don't even have backslash. the backslash only applies in the case we need to concatenate code blocks whose first line is already indented in the previous template. in other words, it only applies to the case of `virtual std::string to_json() const ...`, but in order to have more consistent templates, i didn't incorporate your suggestion in previous revision.